applications/teletype/doc/customer_defined_notification.md

```diff
@@ -0,0 +1,21 @@
+Possible macros
```

Comment:
Could you suggest example how to format documentation for template macros.

Reply:
One suggestions is to break them up by category:

## System
hostname | the system hostname
node | the system node

## Account
...

Reply:
Hello @jamesaimonetti 
Could you look updated macros description